doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1247,23 +1248,24 @@ Here are the syntax elements used by the various notation commands.
              due to legacy notation in the Coq standard library.
              It can be turned on with the ``-w disj-pattern-notation`` flag.
 
+.. note::
+
+   The entry ``strict ident`` parses identifiers as in grammar entry
+   :n:`@ident`. The entry ``name`` parses :n:`@name`, that is
+   identifiers and :g:`_`. The entry ``global`` parses :n:`@qualid`
+   (and thus, as a special case, :n:`@ident`). The entry ``bigint``
+   parses arbitrary large integer numbers as in :n:`@bigint`.
+
 .. note::
 
    As of version 8.13, the entry ``ident`` is a deprecated alias for
```

Comment:
```suggestion
   As of version 8.16, the entry ``ident`` is a deprecated alias for
```

If correct

Reply:
Both 8.13 and 8.15 are incorrect. The deprecation was actually in 8.14.